federation/pkg/dnsprovider/providers/google/clouddns/rrsets.go

```diff
@@ -24,14 +24,24 @@ import (
 	"k8s.io/kubernetes/federation/pkg/dnsprovider/rrstype"
 )
 
-// Compile time check for interface adeherence
+// Compile time check for interface adherence
 var _ dnsprovider.ResourceRecordSets = ResourceRecordSets{}
 
 type ResourceRecordSets struct {
 	zone *Zone
 	impl interfaces.ResourceRecordSetsService
 }
 
+// Compile time check for interface adherence
+var _ dnsprovider.ResourceRecordChangeset = &ResourceRecordChangeset{}
+
+type ResourceRecordChangeset struct {
```

Comment:
nit: Along the lines of previous comment, prefer a separate source file for this type.